src/Microsoft.Diagnostics.Monitoring/Configuration/MonitoringSourceConfiguration.cs

```diff
@@ -19,5 +17,9 @@ public abstract class MonitoringSourceConfiguration
         public const string TplEventSource = "System.Threading.Tasks.TplEventSource";
 
         public abstract IList<EventPipeProvider> GetProviders();
+
+        public virtual bool RequestRundown => true;
```

Comment:
@shirhatti I vaguely remember you debating whether Buffersize or Rundown should be exposed on any of the other calls as a query parameter. What was the outcome of that debate?

Reply:
As discussed in sync up, we'll only expose these options as part of the POST verb.